package-lock.json

```diff
@@ -0,0 +1,8642 @@
+{
```

Comment:
Whoops, you accidentally added a file here (maybe it's from using `npm` instead of `yarn`?). Can you remove this and update the PR?